node/src/components/storage.rs

```diff
@@ -1297,14 +1291,6 @@ impl Storage {
         }
     }
 
-    /// Retrieves block hash by height.
-    fn get_block_hash_by_height(
-        &self,
-        height: u64,
-    ) -> Result<Option<BlockHash>, FatalStorageError> {
-        Ok(self.block_height_index.get(&height).cloned())
-    }
```

Comment:
Yeah, possibly.
I changed that because the header is needed anyway to later get the "era id", but if there's indeed an index for that I can use that. I'll check that.